### PR TITLE
feat: add `constants/float32/min-base10-exponent`

### PR DESCRIPTION
**Pull request by gururaj1512**
Resolves None.

This pull request:

-   Adds implementaion of `constants/float32/min-base10-exponent`

```javascript
var FLOAT32_MIN_BASE10_EXPONENT = require( '@stdlib/constants/float32/min-base10-exponent' );

console.log( FLOAT32_MIN_BASE10_EXPONENT );
// => -38
```

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by Planeshifter** [comment]
@gururaj1512 In the C standard library, the `FLT_MIN_10_EXP` macro for the minimum base 10 exponent for a normal single-precision floating-point number is defined as `-37`.
While the smallest positive normal single-precision floating-point number has a base 10 exponent of `-38` (`~1.17549435e-38`), standard libraries define the minimum base 10 exponent as `-37` to ensure that any number greater than or equal to `10^{-37}` is representable as a normal float. We should use this value instead, as it is the widely accepted and used one in various programming languages.

References:

-   https://en.cppreference.com/w/c/types/limits
-   https://en.cppreference.com/w/cpp/types/numeric_limits/min_exponent10

**Comment by gururaj1512** [comment]
> @gururaj1512 In the C standard library, the `FLT_MIN_10_EXP` macro for the minimum base 10 exponent for a normal single-precision floating-point number is defined as `-37`. While the smallest positive normal single-precision floating-point number has a base 10 exponent of `-38` (`~1.17549435e-38`), standard libraries define the minimum base 10 exponent as `-37` to ensure that any number greater than or equal to `10^{-37}` is representable as a normal float. We should use this value instead, as it is the widely accepted and used one in various programming languages.
> 
> References:
> 
> * https://en.cppreference.com/w/c/types/limits
> * https://en.cppreference.com/w/cpp/types/numeric_limits/min_exponent10

File: lib/node_modules/@stdlib/constants/float32/min-base10-exponent/package.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  ],
+  "keywords": [
+    "stdlib",
+    "stdmath",
+    "constant",
+    "const",
+    "mathematics",
+    "math",
+    "double",
+    "dbl",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/constants/float32/min-base10-exponent/package.json]:
```suggestion
    "single",
```

**Review by Planeshifter — APPROVED** [review]
Thanks for the PR and update, @gururaj1512!

**Pull request merged**